src/Components/CameraFeed/CameraFeed.tsx

```diff
@@ -28,6 +28,8 @@ interface Props {
   shortcutsDisabled?: boolean;
   onMove?: () => void;
   operate: ReturnType<typeof useOperateCamera>["operate"];
+  //Monitor Hide or Show
+  hideMonitorAsset?: boolean;
```

Comment:
1. don't add comments for obvious things
2. what has the word `monitor` got to do over here? How about making the name more appropriate `hideAssetInfo`?

```suggestion
  hideAssetInfo?: boolean;
```